db/migrate/20210328174852_create_moment_templates.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMomentTemplates < ActiveRecord::Migration[6.0]
+  def change
+    create_table :moment_templates do |t|
+      t.string :name
+      t.text :description
+      t.string :slug
+      t.index :slug, unique: true
+      t.integer :user_id
```

Comment:
I think this is preferred to set a foreign key on `user_id`. Base on the factory I saw below, I assume that Moment Templates belonging to a user would be optional?

```suggestion
      t.references :user, foreign_key: true, null: true
```